components/automate-ui/src/app/modules/token/token-list/api-token-list.component.ts

```diff
@@ -139,13 +139,15 @@ export class ApiTokenListComponent implements OnInit, OnDestroy {
     this.store.dispatch(new CreateToken(tok));
 
     const member = stringToMember(`token:${tok.id}`);
-    (this.createTokenForm.controls.policies.value as string[])
```

Comment:
> cannot read property 'forEach' of null

I generally like `.forEach` just doing nothing for an empty array. What do you think, couldn't we do something like
```ts
(this.createTokenForm.controls.policies.value as string[] || []).forEach( id => /* ... */ );
```

It's a matter of code style, though, and what we deem more readible.

Reply:
👍 I do like that... but will add that in a follow-up. I do not want to delay promoting to acceptance with anything non-blocking.